{"id":"../../node_modules/core-js/modules/_iter-call.js","dependencies":[{"name":"D:\\learning-projects\\js\\natours\\package.json","includedInParent":true,"mtime":1738789525091},{"name":"D:\\learning-projects\\js\\natours\\node_modules\\core-js\\package.json","includedInParent":true,"mtime":1738582554313},{"name":"./_an-object","loc":{"line":2,"column":23,"index":85},"parent":"D:\\learning-projects\\js\\natours\\node_modules\\core-js\\modules\\_iter-call.js","resolved":"D:\\learning-projects\\js\\natours\\node_modules\\core-js\\modules\\_an-object.js"}],"generated":{"js":"// call something on iterator step with safe closing on error\nvar anObject = require('./_an-object');\nmodule.exports = function (iterator, fn, value, entries) {\n  try {\n    return entries ? fn(anObject(value)[0], value[1]) : fn(value);\n  // 7.4.6 IteratorClose(iterator, completion)\n  } catch (e) {\n    var ret = iterator['return'];\n    if (ret !== undefined) anObject(ret.call(iterator));\n    throw e;\n  }\n};\n"},"sourceMaps":null,"error":null,"hash":"10a00cc05c7821ae65c973efbba380c5","cacheData":{"env":{}}}